index.js

```diff
@@ -1,46 +1,80 @@
 'use strict'
 
+const _ = require('lodash')
+const fs = require('fs-extra')
 const readPkg = require('read-pkg-up')
 const truncate = require('cli-truncate')
 const wrap = require('wrap-ansi')
 const pad = require('pad')
+const fuse = require('fuse.js')
+const Promise = require('bluebird')
+const homeDir = require('home-dir')
 const types = require('./lib/types')
 
-function getEmojiChoices(types) {
+function getEmojiChoices(types, symbol) {
   const maxNameLength = types.reduce(
     (maxLength, type) => (type.name.length > maxLength ? type.name.length : maxLength
   ), 0)
 
   return types.map(choice => ({
     name: `${pad(choice.name, maxNameLength)}  ${choice.emoji}  ${choice.description}`,
-    value: choice.code
+    value: symbol ? choice.emoji : choice.code
   }))
 }
 
+function loadConfig(res) {
+  let config = _.get(res, 'pkg.config.cz-emoji')
+  if (!config) {
+    try {
+      const czrc = fs.readJsonSync(homeDir('.czrc'))
+      config = czrc.config['cz-emoji']
+    } catch (e) {
+      config = {}
+    }
+
+    config.types = config.types || types
+    config.symbol = config.symbol || false
+    return config
+  }
+}
```

Comment:
Small tweak to get rid of `lodash` and refactor a little. `readPkg` should be part of the `loadConfig` functions (see the bottom of the file).

```js
const defaultConfig = {
  types,
  symbol: false
}
```

```js
function loadConfig() {
  return readPkg()
    .then(({ pkg }) => {
      const config = (pkg && pkg.config && pkg.config['cz-emoji'])
      if (config) return config

      return new Promise((resolve, reject) => {
        fs.readFile(homeDir('.czrc'), 'utf8', (err, content) => {
          if (err) reject(err)
          resolve(JSON.parse(content))
        })
      })
    })
    .then(config => (
      Object.assign({}, defaultConfig, config)
    ))
}
```